dask_expr/_reductions.py

```diff
@@ -472,12 +477,13 @@ def _lower(self):
             split_every=split_every,
             sort=sort,
             shuffle_by_index=getattr(self, "shuffle_by_index", None),
+            shuffle_method=getattr(self, "shuffle_method", None),
         )
 
 
 class Unique(ApplyConcatApply):
-    _parameters = ["frame", "split_every", "split_out"]
-    _defaults = {"split_every": None, "split_out": True}
+    _parameters = ["frame", "split_every", "split_out", "shuffle_method"]
+    _defaults = {"split_every": None, "split_out": True, "shuffle_method": "tasks"}
```

Comment:
Why did you choose `tasks` as default? From what I see, other methods default to `None` and decide at a later point (e.g., during `_lower`).

Reply:
Yeah but I want a different default here, tasks preserves order while disk does not, which is important for drop_duplicates and unique

Reply:
We should probably:
* expand the warnings we raise for `p2p` regarding order stability to also trigger for `disk`
* add comments explaining why we diverge from the standard defaults (otherwise, I can see myself "fixing" this in a few weeks).

Reply:
Fixing would result in a bad experience since tests will complain loudly :)

Agree with the warnings, will do this as a follow up